common/app/views/fragments/commentBox.scala.html

```diff
@@ -4,8 +4,6 @@
     <div class="d-comment-box__meta">
         <span class="d-comment-box__avatar-wrapper"></span>
         <div class="d-comment-box__meta-text">
-            <span class="d-comment-box__author-label">Signed in as</span>
```

Comment:
This information is already displayed in on page load
`<span class="_author_tywwu_16"></span>` which I'm guessing is coming from https://github.com/guardian/discussion-rendering 
S3 `/com-gu-discussion-frontend/CODE/static/discussion-frontend.preact.iife.XXXX.js` 🤔

Reply:
https://github.com/guardian/discussion-frontend probably 😄

Reply:
Thanks, Oliver explained the state of things to me :)